lib/Mojolicious/Plugin/AssetPack/Pipe/Sass.pm

```diff
@@ -64,7 +64,11 @@ sub process {
       my @args = (qw(sass -s), map { ('-I', $_) } @{$opts{include_paths}});
       push @args, '--scss'          if $asset->format eq 'scss';
       push @args, qw(-t compressed) if $attrs->{minified};
-      $self->run(\@args, \$content, \my $css, undef);
+      $self->run(\@args, \$content, \my $css, \my $err);
+      my $exit = $? > 0 ? $? >> 8 : $?;
```

Comment:
Isn't it already catching errors in the run method?
https://github.com/mojolicious/mojo-assetpack/blob/main/lib/Mojolicious/Plugin/AssetPack/Pipe.pm#L28

Reply:
It's `eval or do`, so it only handles cases when the code inside dies, right? It will not handle non-zero return codes, because that is not a runtime error inside eval, do I understand it correctly?

Reply:
In other words, it will handle failures with spawning the command, but I need to handle correct runs with non-zero exit codes, which is not handled by the outside handler. Or at least that's what I think it is doing.